remix.config.js

```diff
@@ -11,6 +11,19 @@ module.exports = {
   serverDependenciesToBundle: [/.*/],
   // appDirectory: "app",
   // assetsBuildDirectory: "public/build",
-  // serverBuildPath: "api/index.js",
+  serverBuildPath: "api/index.js",
   // publicPath: "/build/",
+  // serverDependenciesToBundle: [
```

Comment:
Are we going to need this commented-out section?

Reply:
I don't think so, I'm gonna remove it